nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Migrating to native delivery
+summary: An overview of the tool supporting migrating from timeout manager to native delivery
+reviewed: 2020-05-29
+---
+
+In v7 [native delayed delivery](/nservicebus/messaging/delayed-delivery.md) was introduced across most transports that are supported today.
+Hybrid mode was made available for endpoints, which means that the timeouts that were already registered in the system, would still be consumed by the legacy [TimeoutManager] (/nservicebus/messaging/timeout-manager.md) and the new delayed deliveries  would flow through the native implementation.
+
+Most of the timeouts that were registered through the legacy timeout manager might have been consumed by now. There might be scenarios in which there are timeouts waiting to expire, and those are stored in the timeout storage.
+For those use cases, there is a .NET Core global tool that enables migration of those timeouts to the native delayed delivery infrastructure.
+
+The tool supports live-migration so there's no need to shut down the endpoints before running the tool. The tool will hide the timeouts it will migrate from the legacy TimeoutManager to eliminate duplicate deliveries in the system.
+The tool provides a preview command in order to gather an overview of the endpoints and timeouts to migrate in the system.
+
+## Supported persistence mechanisms
+
+The current version of the tool supports the following persistence mechanisms:
+- [SQL](/persistence/sql/) using SQL Server under the hood
+- [RavenDB](/persistence/ravendb) versions 3 and 4
+
+## Supported transports
+
+The current version of the tool supports the following transports:
+- [RabbitMQ](/transports/rabbitmq/)
+
+## How to install
+
+```
+dotnet tool install migrate-timeouts -g --source
```

Comment:
```suggestion
dotnet tool install migrate-timeouts -g
```

if it's going to be on Nuget publicly available the `--source` argument is not needed.